### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by saadndm** [BridgeSenseMC/Dank-Memer-Grinder#117] [Announcements]
**Feature Difficulty**
:green_square:  Easy
:orange_square:  Medium
:red_square:  Hard
**1.6.0**
~~:red_square: Heist joiner~~
~~:orange_square: Auto adventure~~
~~:orange_square: Auto vote~~
~~:orange_square: Custom search, crime order~~
~~:green_square: Auto scratch~~
~~:green_square: Customizable stream options order~~
**1.7.0**
:orange_square: CLI
**2.0.0 (ideas only)**
:red_square: Command line lightweight version
:red_square: On off schedule
:red_square: Stats tracking
:red_square: Use flows

**Comment by red78massive1573** [comment]
https://github.com/red78massive1573/heistjoiner
that could be an example or a start point on your heist joiner cog

to help you out a bit, this is the embed dict when there is a heist started (on going, not when heist is finished):

```
{'thumbnail': {'url': 'https://cdn.discordapp.com/emojis/969734884039995432.png', 'proxy_url': 'https://images-ext-1.discordapp.net/external/9nL4xUFNPvKE3qiIWDiRaBVzstJ44I03dEmhg3PxYSs/https/cdn.discordapp.com/emojis/969734884039995432.png', 'width': 128, 'height': 128}, 'fields': [], 'color': 2829617, 'type': 'rich', 'description': "They're trying to break into **name**'s bank!\n\nJoin them by clicking `JOIN BANKROB`!", 'title': 'name is starting a bank robbery'}
```

**Reply by saadndm** [reply]
I have exams for the next month but most of them are in the next two weeks so I'm going to be pretty inactive

**Reply by red78massive1573** [reply]
ok

**Reply by dkfjdlkfjdklfdlkfjdkljfkldjfldj** [reply]
> I have exams for the next month but most of them are in the next two weeks so I'm going to be pretty inactive

when will you be introducing new features?

**Comment by ineeddankselfbots** [comment]
what is mean by batch token  input? and any ETA on 1.6 update?
you are doing a wonderful job man i appreciate you

**Reply by red78massive1573** [reply]
batch token input means that you can input many tokens in 1 go and make a new bot accordingly. unlike the current one that needs you to click the + button to make a new bot.

**Reply by ineeddankselfbots** [reply]
oh then what about the striked features in the list will they come or wont come?

**Reply by red78massive1573** [reply]
> oh then what about the striked features in the list will they come or wont come?

they should mean that they are already made.

**Reply by ineeddankselfbots** [reply]
ooh the search order is like we choose the places we want right? like in halloween we can select the place which gives event items right?

**Reply by red78massive1573** [reply]
that should be it
